todo_app/data/trello_items.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, render_template, request
+
+import requests
+import os
+
+def get_trello_cards():
+    url = "https://api.trello.com/1/boards/6205664a19d7b437223061eb/lists" 
```

Comment:
Your .env.template and README mention BOARD_ID but you're not actually using it yet:
Should be something like:
```suggestion
    url = f"https://api.trello.com/1/boards/{os.getenv('BOARD_ID')}/lists" 
```